src/raja/model.cmake

```diff
@@ -1,146 +1,38 @@
-" "")
-
-register_flag_optional(RAJA_IN_PACKAGE
-        "Use if Raja is part of a package dependency:
-        Path to installation" "")
-
-register_flag_optional(CHAI_IN_PACKAGE
-        "Use if CHAI is part of a package dependency:
-        Path to installation. Overrides RAJA path." "")
-
-register_flag_optional(TARGET
-        "Target offload device, implemented values are CPU, NVIDIA, AMD"
-        CPU)
-
-register_flag_optional(CUDA_TOOLKIT_ROOT_DIR
-        "[TARGET==NVIDIA only] Path to the CUDA toolkit directory (e.g `/opt/cuda-11.2`) if the RAJA_ENABLE_CUDA or ENABLE_CUDA flag is specified for RAJA" "")
-
-# XXX CMake 3.18 supports CMAKE_CUDA_ARCHITECTURES/CUDA_ARCHITECTURES but we support older CMakes
-register_flag_optional(CUDA_ARCH
-        "[TARGET==NVIDIA only] Nvidia architecture, will be passed in via `-arch=` (e.g `sm_70`) for nvcc"
-        "")
-
-register_flag_optional(CUDA_EXTRA_FLAGS
-        "[TARGET==NVIDIA only] Additional CUDA flags passed to nvcc, this is appended after `CUDA_ARCH`"
-        "")
-
-register_flag_optional(BLT_DIR
-        "Provide BLT directory, needed for CHAI support on some systems"
-        "")
-
-# compiler vendor and arch specific flags
-set(RAJA_FLAGS_CPU_INTEL -qopt-streaming-stores=always)
+register_flag_optional(MANAGED_ALLOC "Use UM instead of device-only allocation" "OFF")
 
 macro(setup)
-
-
-    if (${TARGET} STREQUAL "CPU")
-        register_definitions(RAJA_TARGET_CPU)
-    elseif (${TARGET} STREQUAL "AMD")
-        register_definitions(RAJA_TARGET_AMD)
-    else ()
-        register_definitions(RAJA_TARGET_NVIDIA)
+    if (POLICY CMP0104)
+        cmake_policy(SET CMP0104 OLD)
     endif ()
 
-    if (EXISTS "${RAJA_IN_TREE}")
-
-        message(STATUS "Building using in-tree RAJA source at `${RAJA_IN_TREE}`")
-
-        set(CMAKE_CXX_STANDARD 14)
-        # don't build anything that isn't the RAJA library itself, by default their cmake def builds everything, whyyy?
-        set(ENABLE_TESTS OFF CACHE BOOL "")
-        set(ENABLE_EXAMPLES OFF CACHE BOOL "")
-        set(ENABLE_REPRODUCERS OFF CACHE BOOL "")
-        set(ENABLE_EXERCISES OFF CACHE BOOL "")
-        set(ENABLE_DOCUMENTATION OFF CACHE BOOL "")
-        set(ENABLE_BENCHMARKS OFF CACHE BOOL "")
-        set(ENABLE_CUDA ${ENABLE_CUDA} CACHE BOOL "" FORCE)
-
-        # RAJA >= v2022.03.0 switched to prefixed variables, we keep the legacy ones for backwards compatibiity
-        set(RAJA_ENABLE_TESTS OFF CACHE BOOL "")
-        set(RAJA_ENABLE_EXAMPLES OFF CACHE BOOL "")
-        set(RAJA_ENABLE_REPRODUCERS OFF CACHE BOOL "")
-        set(RAJA_ENABLE_EXERCISES OFF CACHE BOOL "")
-        set(RAJA_ENABLE_DOCUMENTATION OFF CACHE BOOL "")
-        set(RAJA_ENABLE_BENCHMARKS OFF CACHE BOOL "")
-        set(RAJA_ENABLE_CUDA ${RAJA_ENABLE_CUDA} CACHE BOOL "" FORCE)
-
-        if (ENABLE_CUDA OR RAJA_ENABLE_CUDA)
+    set(CMAKE_CXX_STANDARD 14)
```

Comment:
A couple lines here also need indentation fixes. This file uses 4 spaces per level

Reply:
It's already 4 spaces per level? I don't see the issue? Maybe the diff shows wrong indentation?

Reply:
Ok I see what's going on, I just misread the diff, apologies